cluster/gce/configure-vm.sh

```diff
@@ -762,7 +762,7 @@ users:
 clusters:
 - name: local
   cluster:
-    server: https://kubernetes-master
+    server: https://${1}:443
```

Comment:
> since Vagrant's function is independent, the only call site for this function in the core repo is the one in this file

identical functions with different arg expectations depending on provider makes for *really* hard-to-follow code... these bash scripts are already a maze of twisty passages, all alike.